pkg/controller/controller_binding.go

```diff
@@ -385,6 +401,12 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 			BindResource: &osb.BindResource{AppGUID: &appGUID},
 		}
 
+		if serviceClass.Spec.BindingRetrievable &&
+			utilfeature.DefaultFeatureGate.Enabled(scfeatures.AsyncBindingOperations) {
```

Comment:
Should we log the case where the user has not enabled the feature? Or, how would they know to turn this flag on? Or does it matter?

Reply:
We should at least add some docs explaining the flag (don't see them in this PR). More logs couldn't hurt, but IMO the docs are paramount.